.dockerignore

```diff
@@ -29,3 +29,4 @@ vue/babel.config*
 vue/package.json
 vue/tsconfig.json
 vue/src/utils/openapi
+venv
```

Comment:
i dont really think venv should be ignored because all dependencies are installed in venv when the conainer builds and when we ignore that layer it wont make it into the final image I think ?